lightkurve/targetpixelfile.py

```diff
@@ -1130,6 +1131,124 @@ def cutout(self, center=None, size=5):
         return self.__class__(newfits)
 
 
+    def inspect_pixels(self, ax=None, normalized=False, periodogram=False, aperture_mask=None, style='lightkurve', title=None, **kwargs):
```

Comment:
Regarding the name: I like the idea of having every method that creates a plot start with the prefix `plot_`, so they are easy to discover.  For example, we have `plot_river()` in LightCurve and `plot_diagnostics()` in RegressionCorrector.  Following this philosophy, I wonder if e.g.  `plot_pixels()`, `plot_pixelseries()`, or `plot_pixelgrid()` would be a good name for this feature? What do you think?

Reply:
Good point! I'm in favour of keeping it simple, so how about `plot_pixels()`?